src/components/SearchBar/SearchBar.js

```diff
@@ -6,14 +6,14 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import 'url-search-params-polyfill'
 
 /**
- * Component that renders the searchBar component.
+ * Component that renders the searchBar component, which displays a searchBar in the navigation bar.
```

Comment:
> , which displays a searchBar in the navigation bar.

This is not necessarily true, as the SearchBar component is also used on the RaadpleegHome page

Reply:
Good point, I've changed the component description.